source/products/oab/household/webapp.rst

```diff
@@ -0,0 +1,37 @@
+Webapp
+======
+
+Example Kasko JS embed code (OAB - OAB - Hausrat)
```

Comment:
Why `OAB` x2 here?

Reply:
Double or nothing! ✌️

Reply:
I just copied it from dashboard 🤔 But yeah, I think writing it once makes it clear either way.

Reply:
So there are 2 flows, OAB and Schwarzwalder, that's why 🤷‍♂️